redirects.json

```diff
@@ -0,0 +1,508 @@
+	  "value":"/builders/ethereum/precompiles/utility/eth-mainnet/"
+	},
+	{
+	  "key":"/builders/build/eth-api/debug-trace/",
+	  "value":"/builders/ethereum/json-rpc/debug-trace/"
+	},
+	{
+	  "key":"/builders/build/eth-api/dev-env/truffle/",
+	  "value":"/builders/ethereum/dev-env/"
+	},
+	{
+	  "key":"/builders/build/eth-api/pubsub/",
+	  "value":"/builders/ethereum/json-rpc/pubsub/"
+	},
+	{
+	  "key":"/builders/build/moonbeam-custom-api/",
+	  "value":"/builders/ethereum/json-rpc/moonbeam-custom-api/"
+	},
+	{
+	  "key":"/builders/get-started/eth-compare/index/",
+	  "value":"/learn/core-concepts/"
+	},
+	{
+	  "key":"/builders/get-started/eth-compare/balances/",
+	  "value":"/learn/core-concepts/balances/"
+	},
+	{
+	  "key":"/builders/get-started/eth-compare/consensus-finality/",
+	  "value":"/learn/core-concepts/consensus-finality/"
+	},
+	{
+	  "key":"/builders/get-started/eth-compare/rpc-support/",
+	  "value":"/builders/ethereum/json-rpc/eth-rpc/"
+	},
+	{
+	  "key":"/builders/get-started/eth-compare/security/",
+	  "value":"/learn/core-concepts/security/"
+	},
+	{
+	  "key":"/builders/get-started/eth-compare/transfers-api/",
+	  "value":"/learn/core-concepts/transfers-api/"
+	},
+	{
+	  "key":"/builders/get-started/eth-compare/tx-fees/",
+	  "value":"/learn/core-concepts/tx-fees/"
+	},
+	{
+	  "key":"/builders/get-started/moonbase/",
+	  "value":"/builders/get-started/networks/moonbase/"
+	},
+	{
+	  "key":"/builders/get-started/moonriver/",
+	  "value":"/builders/get-started/networks/moonriver/"
+	},
+	{
+	  "key":"/builders/get-started/moonbeam/",
+	  "value":"/builders/get-started/networks/moonbeam/"
+	},
+	{
+	  "key":"/builders/get-started/networks/layer2/bobabeam/",
+	  "value":"/builders/get-started/networks/"
+	},
+	{
+	  "key":"/builders/interact/hardhat/",
+	  "value":"/builders/ethereum/dev-env/hardhat/"
+	},
+	{
+	  "key":"/builders/interoperability/xcm/xc20/xtokens/",
+	  "value":"/builders/interoperability/xcm/xc20/send-xc20s/"
+	},
+	{
+	  "key":"/builders/interoperability/xcm/fees/",
+	  "value":"/builders/interoperability/xcm/core-concepts/weights-fees/"
+	},
+	{
+	  "key":"/builders/interoperability/xcm/remote-evm-calls/",
+	  "value":"/builders/interoperability/xcm/remote-execution/remote-evm-calls/"
+	},
+	{
+	  "key":"/builders/interoperability/xcm/xc20/mintable-xc20/",
+	  "value":"/builders/interoperability/xcm/xc20/overview/"
+	},
+	{
+	  "key":"/builders/interoperability/xcm/xcm-transactor/",
+	  "value":"/builders/interoperability/xcm/remote-execution/substrate-calls/"
+	},
+	{
+	  "key":"/builders/interoperability/xcm/xc-integration/",
+	  "value":"/builders/interoperability/xcm/xc-registration/"
+	},
+	{
+	  "key":"/builders/pallets-precompiles/precompiles/xcm-utils/",
+	  "value":"/builders/interoperability/xcm/xcm-utils/"
+	},
+	{
+	  "key":"/builders/xcm/fees/",
+	  "value":"/builders/interoperability/xcm/core-concepts/weights-fees/"
+	},
+	{
+	  "key":"/builders/xcm/xc20/index/",
```

Comment:
so this needs to have `/index/` even if that's not in the URL path?

Reply:
Well, that is how we had it in the redirects section of the `mkdocs.yml` file - So, if this was the case, these redirects were wrong all along, right? 

I guess now with server-side we don't need the `/index/` - `/builders/xcm/xc20/` shows a 404

Reply:
It wasn't broken before because the redirects before were based on the file name, not the path it renders in the browser (without `.md`). But anyways, ok cool good to know. Sorry I should have just tested these to confirm myself hahah